front/pages/api/login.ts

```diff
@@ -39,7 +39,7 @@ async function handleMembershipInvite(
     AuthFlowError | SSOEnforcedError
   >
 > {
-  if (membershipInvite.inviteEmail !== user.email) {
+  if (membershipInvite.inviteEmail !== user.email.toLowerCase()) {
```

Comment:
We do it "à la source" for membership invitations and we can't for users cause it would break the login?

Reply:
Nope we could do it as well, since we don't use email for login, but the `sub` property.